hermetic_build/common/model/generation_config.py

```diff
@@ -105,77 +105,77 @@ def __validate(self) -> None:
-
-        new_library = LibraryConfig(
-            api_shortname=__required(library, "api_shortname"),
-            api_description=__required(library, "api_description"),
-            name_pretty=__required(library, "name_pretty"),
-            product_documentation=__required(library, "product_documentation"),
-            gapic_configs=parsed_gapics,
-            library_type=__optional(library, "library_type", "GAPIC_AUTO"),
-            release_level=__optional(library, "release_level", "preview"),
-            api_id=__optional(library, "api_id", None),
-            api_reference=__optional(library, "api_reference", None),
-            codeowner_team=__optional(library, "codeowner_team", None),
-            excluded_poms=__optional(library, "excluded_poms", None),
-            excluded_dependencies=__optional(library, "excluded_dependencies", None),
-            client_documentation=__optional(library, "client_documentation", None),
-            distribution_name=__optional(library, "distribution_name", None),
-            googleapis_commitish=__optional(library, "googleapis_commitish", None),
-            group_id=__optional(library, "group_id", "com.google.cloud"),
-            issue_tracker=__optional(library, "issue_tracker", None),
-            library_name=__optional(library, "library_name", None),
-            rest_documentation=__optional(library, "rest_documentation", None),
-            rpc_documentation=__optional(library, "rpc_documentation", None),
-            cloud_api=__optional(library, "cloud_api", True),
-            requires_billing=__optional(library, "requires_billing", True),
-            extra_versioned_modules=__optional(
-                library, "extra_versioned_modules", None
+        parsed_config = GenerationConfig(
+            googleapis_commitish=_required(
+                config, "googleapis_commitish", REPO_LEVEL_PARAMETER
             ),
-            recommended_package=__optional(library, "recommended_package", None),
-            min_java_version=__optional(library, "min_java_version", None),
-            transport=__optional(library, "transport", None),
+            gapic_generator_version=_optional(config, GAPIC_GENERATOR_VERSION, None),
+            libraries_bom_version=_optional(config, LIBRARIES_BOM_VERSION, None),
+            libraries=parsed_libraries,
         )
-        parsed_libraries.append(new_library)
-
-    parsed_config = GenerationConfig(
-        googleapis_commitish=__required(
-            config, "googleapis_commitish", REPO_LEVEL_PARAMETER
-        ),
-        gapic_generator_version=__optional(config, GAPIC_GENERATOR_VERSION, None),
-        libraries_bom_version=__optional(config, LIBRARIES_BOM_VERSION, None),
-        libraries=parsed_libraries,
-    )
 
-    return parsed_config
+        return parsed_config
 
 
-def __required(config: dict, key: str, level: str = LIBRARY_LEVEL_PARAMETER):
+def _required(config: dict, key: str, level: str = LIBRARY_LEVEL_PARAMETER):
```

Comment:
Thanks for fixing this. Looks like this is recommended by our [style guide](go/pystyle#names-to-avoid). Just curious on any other changes of this kind in this PR?